lotsOfSorting.java

```diff
@@ -14,17 +14,61 @@ public static void main(String[] args) {
 		 * 3. Returns sorted list and time to sort using the chosen sorting alg
 		 */
 
+		boolean exit = false;
+		int input = 0;
+		int a[] = {5,3,4,1,2};
+		
+		displayMenu();
+		
+		do {
+			
+
+			input = numberInput(0, 5);
+
+			if (input == 0) {
+				exit = true;
+			} else if (input == 1) {
+				displayArray(a);
+			} else if (input == 2) {
+				a = shuffle(a);
+			} else if (input == 3) {
+				a = randomSort(a);
```

Comment:
errors out right here

Reply:
resolved in https://github.com/benmwilson/COSC499-Team-GitHub-Exercise/pull/2/commits/4081dde3d933d0032f78ea7a61339c97709c8dd6